.eslintrc

```diff
@@ -8,4 +18,4 @@
     "ecmaVersion": 2021,
     "sourceType": "module"
   }
-}
+}
```

Comment:
please include new lines at the end of all files. it may require a change in the .editorconfig file to do that automatically perhaps